### ISSUE TITLE
Save contrast mode choice in async storage for native

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#3188]
### Describe the Problem
Contrast mode choice should be saved in async storage to keep the theme consistent during the interaction with the app

### Describe Your Preferred Solution

Use async storage to save the state of the contrast mode choice.
